css/style.css

```diff
@@ -0,0 +1,41 @@
+h1,
+h2 {
+  text-align: center;
+}
+
+#main {
+  background-color: white;
+  display: flex;
+}
+
+#title {
+  text-align: center;
+  padding-top: 50px;
+  font-size: x-large;
+  width: 100%;
+}
+
+#image {
+  max-width: 100%;
+  height: 100px;
+  display: block;
+}
+
+#img-caption {
+  background-color: pink;
+  text-align: center;
+  margin-left: 40px;
+  margin-top: 50px;
+  font-size: larger;
+  font-family: Arial, Helvetica, sans-serif;
+  padding: 30px;
+}
+
+#tribute-info {
+  margin-top: 50px;
+  text-align: center;
+  font-size: larger;
+  font-family: Arial, Helvetica, sans-serif;
+  padding: 30px;
+  line-height: 30px;
+}
```

Comment:
Try to group common styles using classes, instead of writing the same code many times. 
Check your CSS file and make sure you have repeated code.